test-data/unit/check-tuples.test

```diff
@@ -1468,7 +1468,7 @@ x9, y9, x10, y10, z5 = *points2, 1, *points2 # E: Contiguous iterable with same
 () = []  # E: can't assign to ()
 
 [case testAssignEmptyBogus]
-() = 1  # E: "Literal[1]?" object is not iterable
```

Comment:
This is a significant change and I don't think that is an improvement. `Literal[1]` is more spefic, I personally like it more.

Reply:
yeah, this is an interesting case. I think it's a different issue that we might need to modify the `format_type`.